### ISSUE TITLE
Zsh history support requires editing /etc/zshrc

### ISSUE THREAD

**Issue by austinjones** [austinjones/tab-rs#30]
`/etc/zshrc` on OSX defines the property:
```
# Save command history
HISTFILE=${ZDOTDIR:-$HOME}/.zsh_history
```

This overwrites the prop set by tab.

**Comment by austinjones** [comment]
I also need a non-zsh solution, but #118 solves the oh-my-zsh case (once I submit the plugin as a PR to oh-my-zsh).

**Comment by austinjones** [comment]
Fixed in #119 - should work without oh-my-zsh too.
